src/go/rdctl/pkg/plist/plist.go

```diff
@@ -25,46 +25,59 @@ import (
 const indentChange = "  "
 
 // convertToPListLines recursively reflects the supplied value into lines for a plist
-func convertToPListLines(v reflect.Value, indent string) ([]string, error) {
-	kind := v.Kind()
+func convertToPListLines(typedValue reflect.Value, actualValue reflect.Value, indent string) ([]string, error) {
```

Comment:
Assuming we end up keeping this as-is, we shouldn't repeat the types:
```suggestion
func convertToPListLines(typedValue, actualValue reflect.Value, indent string) ([]string, error) {
```

Reply:
sure